pkg/collector/corechecks/cluster/kubernetesapiserver/kubernetes_eventbundle.go

```diff
@@ -115,6 +128,14 @@ func (b *kubernetesEventBundle) formatEventText() string {
 	return eventText
 }
 
+func (b *kubernetesEventBundle) fitsEvent(event *v1.Event) (string, bool) {
+	eventText := fmt.Sprintf("**%s**: %s\n", event.Reason, event.Message)
```

Comment:
could you elaborate the format here please ? 
is it the generated final text format for an event that we send ?
is it mocking something ?

on top of that can we just apply `len(event.Reason) + len(event.Message)` ?

Reply:
I see what you mean, then let's keep it as is for now.

I suggest:
- we remove the use of `fmt.Sprintf`
- replace it by some string concat which is way more efficient: `"**" + event.Reason + "**: " + event.Message + "\n"`

Ref: https://github.com/DataDog/datadog-agent/pull/44036